docs/processes/release/tasks/updating-measure-unit.md

```diff
@@ -0,0 +1,128 @@
+# Updating Measure Unit with new CLDR data
+
+This document explains how to update the C and JAVA version of the MeasureUnit
+class with new CLDR data.
+
+Code is generated by running MeasureUnitTest.java unit tests, which writes
+generated code to System.out. Two ways to access this:
+
+1. Within **eclipse**:
+   - Open MeasureUnitTest.java, run it by clicking on the green play button on
+     menu bar.
+   - Copy the generated code from the eclipse console to the clipboard.
+
+2. With **ant**:
+   - Run: `ant checkTest
+     -Dtestclass='com.ibm.icu.dev.test.format.MeasureUnitTest'`
+   - Open the checkTest output: `out/junit-results/checkTest/html/index.html`
+   - Navigate to "System.out" at the bottom of the MeasureUnitTest page to find
+     the generated code, and copy to the clipboard.
+
+After syncing CLDR data with ICU do the following. This documentation assumes
+that you are updating the MeasureUnit clases for ICU 68.
+
+* Check out
+  $GIT_ROOT/icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/MeasureUnitTest.java
```

Comment:
Identify `GIT_ROOT`

Reply:
Will discuss in person when we next chat: I don't know what edit would be an improvement.

Reply:
Deemed "optional feedback", and because this is typically not used in any shell snippets, GIT_ROOT doesn't need to be set/exported in shell. (I considered adding a "export GIT_ROOT=/path/to/icu/source", as I might do for actual shell snippets.)